scenarioo-client/app/components/detailarea/detail-accordion/detail-accordion.component.ts

```diff
@@ -30,18 +33,46 @@ export class DetailAccordionComponent {
     @Input()
     isFirstOpen: boolean;
 
+    @Input()
+    tree: boolean;
+
+    @Input()
+    sketches: boolean;
```

Comment:
name is confusing. Sketches suggest sketches objects but it's a boolean. Maybe has sketches?